src/media.rs

```diff
@@ -56,6 +56,7 @@ const TABLE: &[(&str, Media, &[&str])] = &[
   ("audio/mpeg", Media::Audio, &["mp3"]),
   ("audio/wav", Media::Audio, &["wav"]),
   ("image/apng", Media::Image, &["apng"]),
+  ("image/avif", Media::Image, &["avif"]),
```

Comment:
Let's remove the extension. This will allow inscriptions with the `image/avif` content type to display on explorers, but won't let users create them with `ord wallet inscribe FILE.avif`, since they won't have a recognized extension.

```suggestion
  ("image/avif", Media::Image, &[]),
```

This will cause some tests to break, but they can be fixed by doing this in content_type_for_extension:

```rust
    let mut extensions = TABLE
      .iter()
      .flat_map(|(_, _, extensions)| extensions.first().cloned())
      .collect::<Vec<&str>>();
```